rules/chrome.json

```diff
@@ -1,58 +1,82 @@
 {
   "patterns": [
-    "\\bchrome\\b"
+    "\\bchrome\\b",
+    "\\bchromium\\b"
   ],
   "dependencies": [
+    {
+      "pre_install": [],
+      "packages": ["chromium-browser"],
+      "post_install": [],
+      "constraints": [
+        {
+          "os": "linux",
+          "distribution": "ubuntu",
+          "versions": ["20.04", "22.04"]
```

Comment:
Is Ubuntu 24.04 supported?

Reply:
Yes. Good catch, I lost it in the refactoring as starting with 24.04, the key is `chromium` instead of `chromium-browser`. Added it.